### DISCUSSION TITLE
请问 Antd 内组件总览是如何实现的?

### DISCUSSION THREAD

**Discussion by Silence-dream** [ant-design/ant-design#41435] [Q&A] [answered]
我注意到源码里面调用了

```md
<ComponentOverview></ComponentOverview>
```
即可渲染出全部的组件，这一个功能是在哪里实现的？

![image](https://user-images.githubusercontent.com/42824008/227441109-8573ec44-3fba-4ae9-9024-28d811acf9da.png)

https://github.com/ant-design/ant-design/blob/7b581e7ef7332efc2e423ec63e02ebf77baa7874/components/overview/index.zh-CN.md?plain=1#L8

**Comment by li-jia-nan** [comment, marked as answer]
`.dumi/theme/builtins/ComponentOverview/index.tsx`

**Comment by yoyo837** [comment]
天, 他vscode窗口好炫! 🤩

**Reply by Silence-dream** [reply]
我是webstorm😄

**Reply by yoyo837** [reply]
<img width="141" alt="image" src="https://user-images.githubusercontent.com/6134547/227442198-03333dbf-50f2-4219-aad4-3c061f456ec3.png">
